test/e2e/plain-v0_provisioner_test.go

```diff
@@ -22,6 +22,10 @@ const (
 	defaultSystemNamespace = "rukpak-system"
 )
 
+var (
+	ctx = context.Background()
+)
```

Comment:
Nit/question: This _probably_ doesn't matter, but I wonder if this could ever cause problems as a global, which gets used across all tests.  WDYT about initializing `ctx` in a `BeforeEach` in the root `Describe`?

```go 
var _ = Describe("root", func() {
 	var ctx context.Context
	BeforeEach(func() {
		ctx = context.Background()
	})
})
```

Reply:
It's probably worth eliminating any global state that's not needed so I updated to instantiate this variable in each top-level BeforeEach.